netsuite/10.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc

```diff
@@ -10,96 +10,123 @@ Upgrade NetSuite Connector to version 10.0 to use the new features.
 |9.3.0 |10.0.0
 |===
 
-== Changes in This Release
+== New Features and Changes in This Release
 
-This release contains the following changes:
+This release contains the following new features:
 
-* These authentication types were removed:
-** *Login Authentication* 
-** *SSO Authentication* 
 * These operations were added: 
```

Comment:
You want to omit this as a first-level bullet and promote the subbullets, right?
```suggestion
* *Get Account Governance Info* 
* *Get Integration Governance Info* 
```
We are calling these new features rather than new operations?

Reply:
Adding new operations is considered a feature. New operations = new features.